kitchen.yml

```diff
@@ -75,3 +78,38 @@ suites:
           hostnames: slave
     excludes:
       - centos-6
+      - centos-8
+  - name: multi-node-8
+    driver:
+      name: terraform
+      command_timeout: 1800
+      variables: 
+          centos_image: 'CentOS 8.2'
+    provisioner: 
+      name: terraform
+      variables: 
+          centos_image: "CentOS 8.2"
```

Comment:
Feels like this is duplicated. Is this really needed for both driver and provisioner?

Reply:
oops, good catch!